Scheduler.py

```diff
@@ -1,43 +0,0 @@
-from apscheduler.schedulers.blocking import BlockingScheduler
-import datetime
-import csv
-from datetime import timedelta
-import os
-#TEST
-
-def mlb():
-    os.system('python MLBSpider.py')
-
-
-def fangraph():
-    os.system('python FanGraphSpider.py')
-
-
-# Reads first game start time from csv and schedules FanGraphSpider to begin running at that time.
-def schedulescheduler():
-    start_time = ''
-    with open('schedule.csv', 'r') as input:
-        reader = csv.reader(input)
-        for line in reader:
-            start_time = line[1]
-            break

```

Comment:
This loop will only run once because of this break. If that's the intent and you just want to set the start time from the first line, you can replace the loop with
```
...
reader = csv.reader(input)
start_time = next(reader)[1]
```
Think of code as communicating intent as well as actually instructing the computer, so if your intent is to get one line, this code is more clear. Even if you're the only one that ever works on this, it can help you remember what the hell you were doing if you don't look at the code for a month.